### DISCUSSION TITLE
希望自动播放下一集的功能只作用于多P合集内

### DISCUSSION THREAD

**Discussion by wialny** [the1812/Bilibili-Evolved#1647] [功能建议 / Ideas]
最近不知道为什么, 不管是多P还是单P都会自动播放. 以前明明只有合集才会自动播放的
所以希望加入能把单P和多P分开设置的自动播放功能

**Comment by Geeyun-JY3** [comment]
重复：#1634

**Comment by Geeyun-JY3** [comment]
见：https://github.com/the1812/Bilibili-Evolved/discussions/1634#discussioncomment-593169
